css/properties/column-gap.json

```diff
@@ -208,10 +208,19 @@
                 "version_added": "10.3",
                 "alternative_name": "grid-gap"
               },
-              "samsunginternet_android": {
-                "version_added": "6.0",
-                "alternative_name": "grid-gap"
-              },
+              "samsunginternet_android": [
+                {
+                  "version_added": "9.0"
+                },
+                {
+                  "alternative_name": "grid-gap",
+                  "version_added": "7.0"
+                },
```

Comment:
This will need a partial reversion we implemented CSS Grid earlier than our Chrome Engine would indicate.

We implemented in 6.0 not 7.0.